src/views/app/detail-panel/preview/file-preview.tsx

```diff
@@ -15,6 +17,24 @@ export const humanFileSize = (inputSize: number): string => {
 	return `${(inputSize / 1024 ** i).toFixed(2).toString()} ${['B', 'KB', 'MB', 'GB', 'TB'][i]}`;
 };
 
-export const previewType = (contentType: string): string | undefined =>
-	// eslint-disable-next-line no-nested-ternary
-	contentType?.startsWith('image') ? 'image' : contentType?.endsWith('pdf') ? 'pdf' : undefined;
+export const previewType = (contentType: string): string | undefined => {
+	if (contentType?.startsWith('image') && !contentType?.includes('photoshop')) return 'image';
+	if (
+		contentType?.endsWith('pdf') ||
+		includes(
+			[
+				'text/csv',
+				'text/plain',
+				'application/vnd.ms-excel',
+				'application/msword',
+				'application/vnd.oasis.opendocument.spreadsheet',
+				'application/vnd.oasis.opendocument.presentation',
+				'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+				'application/vnd.oasis.opendocument.text'
+			],
+			contentType
+		)
+	)
```

Comment:
why pdf is return for documents ?

Reply:
As documents api is returning documents in pdf format.